applicationinsights-react-native/src/ReactNativePlugin.ts

```diff
@@ -39,6 +44,8 @@ export class ReactNativePlugin extends BaseTelemetryPlugin {
         let _waitingTimer: ITimerHandler;
         let _waitingItems: { item: ITelemetryItem, itemCtx?: IProcessTelemetryContext }[] = null;
         let _deviceInfoModule: IDeviceInfoModule;
+        let notInit = true;
```

Comment:
These should be set in the `_initDefaults()` as when the component gets unloaded these defaults should be restored.

So we still need to define the properties here, but with no value.